src/nvim/memline.c

```diff
@@ -1767,7 +1767,7 @@ ml_get_buf (
    * Don't use the last used line when 'swapfile' is reset, need to load all
    * blocks.
    */
-  if (buf->b_ml.ml_line_lnum != lnum || mf_dont_release) {
+  if (buf->b_ml.ml_line_lnum != lnum) {
```

Comment:
We never release blocks from memory anymore (in practice it never happened) since the memory limits are never reached. The function used to release blocks was removed.

The `getlines` block doesn't even make a lot of sense anymore. It kept because of
https://github.com/neovim/neovim/pull/4486/files#r57671893